model/nc_news.models.js

```diff
@@ -20,4 +20,19 @@ function selectTopics() {
 	});
 }
 
-module.exports = { selectTopics, selectArticleById };
+function fetchArticles() {
+	return db
+		.query(
+            `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.article_img_url, articles.votes,
+            COUNT(comments.comment_id)::INT AS comment_count 
+            FROM articles LEFT JOIN comments 
+            ON articles.article_id = comments.article_id
+            GROUP BY articles.article_id
+            ORDER BY articles.created_at DESC;`
```

Comment:
great job - so we're giving it an order we just need to check that its working in our tests